src/rnp/rnpcfg.c

```diff
@@ -75,7 +75,7 @@ rnp_cfg_load_defaults(rnp_cfg_t *cfg)
     rnp_cfg_setstr(cfg, CFG_HASH, DEFAULT_HASH_ALG);
     rnp_cfg_setint(cfg, CFG_ZALG, PGP_C_ZIP);
     rnp_cfg_setint(cfg, CFG_ZLEVEL, 6);
-    rnp_cfg_setstr(cfg, CFG_CIPHER, "cast5");
+    rnp_cfg_setstr(cfg, CFG_CIPHER, "aes128");
```

Comment:
LGTM but ideally this would be a define somewhere. We could use a `defaults.h` or something to centralize these things. FFI actually defaults to AES-256 -- if we add a define then we can update FFI to use the same here:
https://github.com/riboseinc/rnp/blob/ce5f4bcd2df94ab2dd08c9153c31c20254989f43/src/lib/rnp2.c#L1098

(Though some of this stuff will change when we have an actual preferences framework in place anyways)

Reply:
@dewyatt Agree, I had a bunch of other constants which I didn't know where to put. Will update PR soon with defaults.h